client/lib/faux-plugins.js

```diff
@@ -207,6 +209,18 @@ module.exports = function(roomName) {
       }
     })
 
+    var parseYoutubeTime = function(time) {
+      var timeReg = /([0-9]+h)?([0-9]+m)?([0-9]+s?)?/
+      var match   = time.match(timeReg)
```

Comment:
Please don't line up equals signs -- it makes code harder to maintain and encourages future whitespace-only changes.